### PR TITLE
[liveme] API update

### PR DESCRIPTION
**Pull request by back-to**
closes streamlink/streamlink#1295

### CODE REVIEW

File: src/streamlink/plugins/liveme.py [hunk 1]
```diff
@@ -26,6 +27,16 @@ class LiveMe(Plugin):
     def can_handle_url(cls, url):
         return cls.url_re.match(url) is not None
 
+    def _random_t(self, t):
+        n = "ABCDEFGHJKMNPQRSTWXYZabcdefhijkmnprstwxyz2345678"
+        e = len(n)
+        a = ""
+        r = 0
+        while (r < t):
+            a += n[math.floor(random.randrange(e))]
```
Comment (beardypig) [inline src/streamlink/plugins/liveme.py]: `random.choice` might be a better method to use, eg. `random.choice(n)`. 
Personally, I'd just use a list comprehension/generator to do it :) 
```python
def _random_t(self, t):
    return "".join(random.choice("ABCDEFGHJKMNPQRSTWXYZabcdefhijkmnprstwxyz2345678") for _ in range(t))
```

**Comment by beardypig** [comment]
Nice to see you back to streamlink @back-to ... :)

File: src/streamlink/plugins/liveme.py (hunk 1 above)
Comment (back-to) [reply src/streamlink/plugins/liveme.py]: i just did the same, as it is on their website.

```js
function t(t) {
    t = t || 32;
    for (var n = "ABCDEFGHJKMNPQRSTWXYZabcdefhijkmnprstwxyz2345678", e = n.length, A = "", r = 0; r < t; r++) A += n.charAt(Math.floor(Math.random() * e));
    return A
}
```

but `random.choice` looks easier.

**Comment by gravyboat** [comment]
Looks good, thanks @back-to and thanks @beardypig for the review.

**Pull request merged**